corgi/tasks/brew.py

```diff
@@ -118,16 +128,20 @@ def slow_fetch_brew_build(build_id: int, save_product: bool = True, force_proces
                 slow_load_errata.delay(e, force_process=force_process)
 
     build_ids = component.get("nested_builds", ())
-    logger.info("Fetching brew builds for %s", build_ids)
+    logger.info("Fetching brew builds for (%s, %s)", build_ids, b_type)
     for b_id in build_ids:
-        logger.info("Requesting fetch of nested build: %s", b_id)
-        slow_fetch_brew_build.delay(b_id, save_product=save_product, force_process=force_process)
+        logger.info("Requesting fetch of nested build: (%s, %s)", b_id, b_type)
+        slow_fetch_brew_build.delay(
+            b_id, b_type, save_product=save_product, force_process=force_process
+        )
 
-    logger.info("Requesting software composition analysis for %s", build_id)
+    logger.info("Requesting software composition analysis for %s", softwarebuild.uuid)
     if settings.SCA_ENABLED:
-        cpu_software_composition_analysis.delay(build_id, force_process=force_process)
+        cpu_software_composition_analysis.delay(
+            str(softwarebuild.uuid), force_process=force_process
+        )
```

Comment:
Do we also need to pass the b_type value to the SCA task?
```suggestion
        cpu_software_composition_analysis.delay(
            str(softwarebuild.uuid), b_type, force_process=force_process
        )
```

Reply:
No need, we lookup the the SoftwareBuild there by unique UUID, not id/type combination.